modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/edit_asset_list_entry.jsp

```diff
@@ -31,7 +31,9 @@ portletDisplay.setURLBack(redirect);
 renderResponse.setTitle(assetListDisplayContext.getAssetListEntryTitle());
 %>
 
-<div class="container-fluid container-fluid-max-xl container-view">
+<clay:container
+	className="c-py-4"
```

Comment:
Why change `container-view` (semantic), by `c-py-4`, presentation?

I know we did something like this for `form-container-lg`, but I'm not sure this helps in the long run. Since now, people can't rely on a common definition of how `container-view` works, but they need to know exactly which padding to apply... 

If this is something you and @marcoscv-work and @pat270 have agreed on, then I'd expect at least to see `.container-view` go away from our CSS alleviating the weight. 

I'd prefer to stick with common classes (if they are indeed common) rather than with cryptic specific utils that don't have meaning... but if you're all on board with this we can see where it takes us :)

/cc @wincent

Reply:
I changed it because we are already using several utility class (most of them are not from clay and have the `!important` attribute) and this is what `.container-view` https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/variables/_globals.scss#L80 and `.container-form-lg` do 
https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/variables/_globals.scss#L68 I was hoping to remove those classes if we don't need them

Reply:
Seeing the PR, I using utilities feels like an anti-pattern and contrary to a design system. It will complicate any changes we want to do in the future in an even harder way.

At some point, if we want to, let's say, make these have a padding of 6... how can we tell a `c-py-4` apart from another one? It should be easy to do so on a `.container-view`, but since we lose semantics by using utilities, this will no longer be the case...

If we care about the `!important` attribute, then maybe we should remove it from `.container-view`... and if we care about consistency and repetition, maybe we could have `.container-view @extend .c-py-4`?

Reply:
If you prefer I can change it back to
- `c-py-3 c-py-md-5` (and variations) > `container-form-lg`
- `c-py-4` > `container-view`

This seemed the perfect opportunity to get rid of them 😄

Reply:
I agree with @jbalsas utilities are bad for Portal. We are shooting ourselves in the foot with this. If we really want to use utilities we should provide a semantic class name that does nothing at the very least.

Reply:
> This seemed the perfect opportunity to get rid of them

So... indulge my little detour...

It is our [stated mission](https://docs.google.com/document/d/12Y5WQTkJjEKqU5TQoESZ-FrCkLI3XPnr1ouB_npCaYY) as an infra team:

> To help, support and guide Liferay teams to succeed and deliver robust, safe, performant and dynamic experiences in time and with high quality

So, always weigh in and balance those principles when making decisions.

In this case, my reasoning would be:
- Removing an unnecessary class that's already there. Good for performance
- Changing `container-form-lg` with `c-py-3 c-py-md-5` makes this harder to understand and remember so developers will likely have a harder time knowing what to do. Bad for consistency and speed.
- Changing to such utilities is also more error prone because writing `c-py-3 c-py-md-5` is harder than `container-form-lg` and it's also harder to detect errors. Bad for safety and robustness.

So, based on the above I'd say: 
- Let's keep the `container-form-lg` class to push this forward. This change has enough value in itself in terms of consistency, safety and delivery speed to stand on its own.
- Once we're done, let's check why we need that class. Maybe it's not that we need `c-py-3...` it's that something somewhere else needs adjusted. That would be a better fix imho.
- Once done with this one... let's do a similar thing with the rest of `className` usages in `clay:container`, `clay:row` and `clay:col`... there's probably lessons to be learned there!